ophyd/signal.py

```diff
@@ -810,6 +810,45 @@ def limits(self):
         return (self._metadata['lower_ctrl_limit'],
                 self._metadata['upper_ctrl_limit'])
 
+    def get_or_timeout(self, *, as_string=None, form='time', 
```

Comment:
I don't think we want to add a new public function.  

The `if info is None:` make `get` already have the semantics of `get_or_timeout`.  We might as well put this code in-line in `get` and remove the `if info is None` block right below.

Reply:
ok